packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,60 @@
+
+.navMenuContainer {
+    display: flex;
+    flex-direction: column;
+    padding: 36px var(--space-base) 0 var(--space-base);
```

Comment:
I think we need to move the left and right padding off of the parent and onto the children - this approach "traps" the scrollbar inside of the nav items in a way that overlaps, and clips the edges of the focus states:
![image](https://github.com/user-attachments/assets/39691b5a-8b8b-4ba9-b7b2-e82a7d328acf)

Moving the left and right padding to the children gives the scrollbar some room to breathe and leaves the focus states unclipped:
![image](https://github.com/user-attachments/assets/67ef0db8-95c3-40c4-ac0a-a78d20c6930c)

Reply:
You seem to want things in a very specific way, and it is just adding time going back and forth through these comments, so maybe it would be better if you wanted to make the changes on my branch directly? I don't mind!

Reply:
Sure! I can push a few things up 👍